.gitlab-ci.yml

```
@@ -138,6 +142,8 @@ test:licenses:
       junit:
         - junit/results.xml
     when: always
+  tags:
+    - hetzner-amd-beefy-privileged
```

The privileged runner is currently using `privileged=true` and the docker executor, hence is critical and should be used for internal jobs or protected branches... [See the references](https://docs.gitlab.com/runner/security/#usage-of-docker-executor).

Suggestion: keep using a standard GCP runner, and switch to Hetzner when we can find a suitable alternative (https://northerntech.atlassian.net/browse/QA-739)

```suggestion
    - mender-qa-worker-generic-light
```